### PR TITLE
feat(kuma-cp) create default traffic permission for Mesh

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
### Summary

Traffic Permission is created implicitly per mesh so when mTLS is enabled, everything still works the same

### Documentation

- [ ] TODO [documentation PR](https://github.com/kumahq/kuma-website/pull/XXX)

### CODE REVIEW

**Review by nickolaev — APPROVED** [review]
Looks perfectly fine to me.

File: pkg/plugins/runtime/k8s/controllers/mesh_defaults_controller.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+func (r *MeshDefaultsReconciler) Reconcile(req kube_ctrl.Request) (kube_ctrl.Result, error) {
+	log := r.Log.WithValues("mesh", req.NamespacedName)
+
+	// we skip default name Mesh because pkg/defaults/mesh.go uses ResourceManager that executes CreateDefaultMeshResources already
+	if req.Name == model.DefaultMesh {
+		return kube_ctrl.Result{}, nil
+	}
+
+	if err := defaults_mesh.CreateDefaultMeshResources(r.ResourceManager, req.Name); err != nil {
+		log.Error(err, "could not create default mesh resources")
```
Comment (lobkovilya) [inline pkg/plugins/runtime/k8s/controllers/mesh_defaults_controller.go:33]: What if it's gonna fail here? If I understand it right it reconciles only on Create? Probably we need to return here:
```go
return kube_ctrl.Result{
    Requeue: true,
}, err
```
to run it again?

Comment (jakubdyszkiewicz) [reply pkg/plugins/runtime/k8s/controllers/mesh_defaults_controller.go]: That's a good point. I should return here an error. Then Kubernetes does the retry with exponential backoff for us.

Comment (jakubdyszkiewicz) [reply pkg/plugins/runtime/k8s/controllers/mesh_defaults_controller.go]: (I tested it, even with create only - it does the retry)

**Review by lobkovilya — APPROVED** [review]

**Pull request merged**